### PR TITLE
sci-electronics/librepcb: add 0.1.0 release, remove old 0.1.0_p201805

### PR DESCRIPTION
**Pull request by ktrace**
- add release 0.1.0
- change http to https in ebuilds
- fix copyright header
- remove old version
- close PR https://github.com/gentoo/gentoo/pull/10345 as unneeded

### CODE REVIEW

**Comment by Petross404** [comment]
Excellent news. I am back to my Gentoo PC, let me know if there is anything I can do.

File: sci-electronics/librepcb/librepcb-0.1.0.ebuild [hunk 1]
```diff
@@ -1,41 +1,34 @@
...
-EAPI=6
+EAPI=7
 
 inherit qmake-utils
 
 DESCRIPTION="Free EDA software to develop printed circuit boards"
 HOMEPAGE="https://librepcb.org/"
-MY_P="LibrePCB-first_pcb"
-SRC_URI="https://github.com/LibrePCB/LibrePCB/archive/first_pcb.tar.gz -> ${MY_P}.tar.gz"
+#MY_P="LibrePCB-${PV}"
```
Comment (a17r) [inline sci-electronics/librepcb/librepcb-0.1.0.ebuild]: if the commented code is unnecessary, please remove it.

Comment (ktrace) [reply sci-electronics/librepcb/librepcb-0.1.0.ebuild]: done

**Comment by a17r** [comment]
thanks!

**Pull request closed without merging**